nuxeo-compound-documents/src/test/resources/operations-test-contrib.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.compound.documents.operations.customisation.test" version="1.0">
+  <require>org.nuxeo.compound.documents.operations</require>
+  <require>org.nuxeo.compound.documents.types</require>
+  <extension target="org.nuxeo.automation.scripting.internals.AutomationScriptingComponent" point="operation">
+    <scriptedOperation id="javascript.GetCompoundDocumentType">
+      <inputType>document</inputType>
+      <outputType>string</outputType>
+      <category>javascript</category>
+      <param name="entries" type="strings" />
+      <script>
+        function run(input, params) {
+          for (var i = 0; i &lt; params.entries.length ; i++) {
```

Comment:
Can't we get rid of `&lt;` by replacing it with `<` if the whole script is surrounded with `<![CDATA[`?

Reply:
Oh ok so this is why some scripts are surrounded like so. Had no idea.